src/main/scala/com/philcoutinho/gold/CommentsInTime.scala

```diff
@@ -0,0 +1,16 @@
+package com.philcoutinho.gold
+
+
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+
+object CommentsInTime {
+
+  def commentsintime(CommentData : DataFrame, spark : SparkSession):DataFrame={
+
+    CommentData.createOrReplaceTempView("my_table")
+    val result = spark.sql("SELECT __typename, created_at, comment_id, owner_id, comment_username, text " +
+      " FROM my_table WHERE FROM_UNIXTIME (created_at) BETWEEN ('2020-01-01') AND ('2021-05-16')")
```

Comment:
add the time as arguments in the function commentsintime

Reply:
Done